setup.cfg

```diff
@@ -57,6 +57,7 @@ test =
     coverage
     skyfield>=1.20
     sgp4>=2.3
+    packaging
```

Comment:
Shouldn't packaging be a runtime requirement now, like `numpy` and `pyerfa`?

Reply:
I didn't think so because it is only imported in the tests. Or at least that was the goal -- using `packaging` in the tests but not in the main package to avoid adding a new dependency.

Reply:
Aha, I hadn't realized this would be just for testing - in that case we actually do not need to add a requirement at all, since `pytest` itself already requires packaging! (It must do its version tests in, e.g., `pytest.importorskip` with it...).